src/display.jl

```diff
@@ -48,17 +48,17 @@ function backend_display(::Missing, ::Scene)
     """)
 end
 
-Base.display(fap::FigureAxisPlot) = display(fap.figure)
-Base.display(fig::Figure) = display(fig.scene)
+Base.display(fap::FigureAxisPlot; kw...) = display(fap.figure; kw...)
+Base.display(fig::Figure; kw...) = display(fig.scene; kw...)
 
-function Base.display(scene::Scene)
+function Base.display(scene::Scene; update=true)
 
     if !use_display[]
         return Core.invoke(display, Tuple{Any}, scene)
     else
         # set update to true, without triggering an event
         # this just indicates, that now we may update on e.g. resize
```

Comment:
is this still correct?

Reply:
It think not since `update!` is in an event loop: https://github.com/JuliaPlots/Makie.jl/blob/33e600665a48f4a26924cdd3fd0c7471379f039d/src/scenes.jl#L115

Should we remove ?